<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 149, 148);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>20</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/FI.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>510</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/UAT.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetAlarm">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
    </property>
    <property name="text">
     <string>Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelAlarm">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_alarm">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 223, 187);
 border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
